go/consensus/tendermint/seed/seed.go

```diff
@@ -83,6 +95,106 @@ func (srv *seedService) Cleanup() {
 	// No cleanup in particular.
 }
 
+// Synced returns a channel that is closed once synchronization is complete.
+func (srv *seedService) Synced() <-chan struct{} {
+	// Seed is always considered synced.
+	ch := make(chan struct{})
+	close(ch)
+	return ch
+}
+
+func (srv *seedService) GetStatus(ctx context.Context) (*consensus.Status, error) {
+	status := &consensus.Status{
+		ConsensusVersion: version.ConsensusProtocol.String(),
+		Backend:          api.BackendName,
+		Features:         srv.SupportedFeatures(),
+	}
+
+	// List of consensus peers.
+	tmpeers := srv.p2pSwitch.Peers().List()
+	peers := make([]string, 0, len(tmpeers))
+	for _, tmpeer := range tmpeers {
+		p := string(tmpeer.ID()) + "@" + tmpeer.RemoteAddr().String()
+		peers = append(peers, p)
+	}
+	status.NodePeers = peers
+
+	return status, nil
+}
+
+// Unimplemented consensus Backend methods.
```

Comment:
The only missing methods are now the backends (epochtime, registry, beacon...) and `ConsensusKey` (which is only used in tests anyway). 

If we want, we could implement those remaining methods here (can still just panic on those), and get rid of the `consensus/api.BaseBackend` since it's not used anywhere else.

cc @kostko

Reply:
Agreed, implement the methods and remove the base backend.